lib/hanikamu/concerns/form.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require "active_model"
+
+module Hanikamu
+  # :nodoc
+  module Form
+    extend ActiveSupport::Concern
+
+    Error = Class.new(Hanikamu::Service::Error) do
+      attr_reader :form
+
+      def initialize(form)
+        @form = form
+
+        super(form.errors.full_messages.join(", "))
+      end
+    end
+
+    included do
+      include ActiveModel::Validations
+    end
+
+    # this is not right
+    class_methods do
+      def call!(options = {})
```

Comment:
I want this method to first perform validations and then call super on the Class that it is included in